packages/medusa/src/api-v2/admin/sales-channels/middlewares.ts

```diff
@@ -56,4 +57,9 @@ export const adminSalesChannelRoutesMiddlewares: MiddlewareRoute[] = [
     matcher: "/admin/sales-channels/:id",
     middlewares: [],
   },
+  {
+    method: ["POST"],
+    matcher: "/admin/sales-channels/:id/products/batch/add",
+    middlewares: [transformBody(AdminPostSalesChannelsChannelProductsBatchReq)],
```

Comment:
discussion: Same question as another pr, should we support transformQuery here wdyt?